README.md

```diff
@@ -1 +1,35 @@
-# node-debugger-lab
\ No newline at end of file
+# Debug a Script Lab
+
+## Objectives
+
+1. Use Node built-in debugger by putting statements into a given script to find a problem by using debugger
+2. Launch debugger
+3. Fix the script
+
+
+
+## Introduction
+
```

Comment:
this lab seems to deal with creating a bank account and reading amount in account. Add that to the intro...something like "You've been hired by XYZ bank to fix some of their account creation code. They had some bad developers in the past that left some bugs in the code. Your job is to fix all the issues and get the code functional again."